File: leetcode/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        count = 1
        curr = nums[0]
        prev = curr

        for i in range(1, len(nums)):
            prev = curr
            curr = nums[i]

            if prev != curr :
                nums[count] = curr
                count+=1
        
        return count

        

        

File: leetcode/26-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        int count = 1, curr = nums[0], prev;

        for( int i =1; i< nums.size(); i++)
        {
            prev = curr;
            curr = nums[i]; 

            if( prev != curr)
            {
                nums[count++] = curr;
            }
        }

        return count;
    }
};